benchmarks/python/conftest.py

```diff
@@ -28,6 +28,20 @@ def pytest_addoption(parser):
         help="Benchmarks torch.compile mode.",
     )
 
+    parser.addoption(
+        "--rounds",
+        action="store",
+        default=10,
+        help="Number of rounds for each benchmark.",
+    )
+
+    parser.addoption(
+        "--warmup-rounds",
+        action="store",
+        default=1,
+        help="Number of warmup rounds for each benchmark.",
+    )
```

Comment:
I think these options should be prefixed with `benchmark-`, like `--benchmark-rounds`. This matches the other default options from pytest-benchmark such as `--benchmark-min-rounds`.

Reply:
Great point!